.travis.yml

```diff
@@ -17,7 +17,8 @@ after_success:
 # Linting and Integration tests need to run first to reset the PR build status to pending.
 stages:
   - 'Linting'
-  - 'Integration tests'
+  - 'Integration Tests'
```

Comment:
travis gui will lowercase this for you even though you specify upppercase... it's annoying :(

Reply:
yeah realized, will change it.